mm/alignment.py

```diff
@@ -391,6 +391,114 @@ def hash(cls, kwargs):
         )
 
 
+class PlotAlignmentJob(Job):
+    """
+    Plots an alignment from its log file.
+    This is an isolate job based on the tasks available in :func:`i6_core.mm.alignment.AlignmentJob.plot`
+    and :func:`i6_core.corpus.filter.FilterSegmentsByAlignmentConfidenceJob.plot`.
+    """
+
+    def __init__(
+        self,
+        alignment_log_files: List[tk.Path],
+        clip_low: Optional[int] = None,
+        clip_high: Optional[int] = None,
+        clip_percentile_low: Optional[int] = None,
+        clip_percentile_high: Optional[int] = None,
+        zoom_x_min: Optional[float] = None,
+        zoom_x_max: Optional[float] = None,
+        zoom_y_min: Optional[float] = None,
+        zoom_y_max: Optional[float] = None,
+        num_bins: int = 50,
+    ):
+        """
+        :param alignment_log_files: Alignment log files from the alignment job.
+        :param clip_low: Number symbolizing the absolute number at which the plot will be clipped to the left.
+            If given along with any other value restricting the minimum, the most restrictive value will prevail.
+            By default clips at the minimum value. Has priority over :param:`clip_percentile_low`.
+        :param clip_high: Number symbolizing the absolute number at which the plot will be clipped to the right.
+            If given along with any other value restricting the maximum, the most restrictive value will prevail.
+            By default clips at the maximum value. Has priority over :param:`clip_percentile_high`.
+        :param clip_percentile_low: Number symbolizing the percentile at which the plot will be clipped to the left.
+            If given along with any other value restricting the minimum, the most restrictive value will prevail.
+            By default clips at the minimum value.
+        :param clip_percentile_high: Number symbolizing the absolute number at which the plot will be clipped to the right.
+            If given along with any other value restricting the maximum, the most restrictive value will prevail.
+            By default clips at the maximum value.
+        :param zoom_x_min: Minimum X value in which to zoom in on the plot. If `None`, won't zoom in.
+        :param zoom_x_max: Maximum X value in which to zoom in on the plot. If `None`, won't zoom in.
+        :param zoom_y_min: Minimum Y value in which to zoom in on the plot. If `None`, won't zoom in.
+        :param zoom_y_max: Maximum Y value in which to zoom in on the plot. If `None`, won't zoom in.
+        :param num_bins: Number of histogram bins. By default `50`.
+        """
+        self.alignment_log_files = alignment_log_files
+        self.clip_low = clip_low
+        self.clip_high = clip_high
+        self.clip_percentile_low = clip_percentile_low
+        self.clip_percentile_high = clip_percentile_high
```

Comment:
```suggestion
        assert clip_low < clip_high
        self.clip_percentile_low = clip_percentile_low
        assert 0.0 <= clip_percentile_low <= 100.0, "percentile should be between 0 and 100"
        self.clip_percentile_high = clip_percentile_high
        assert 0.0 <= clip_percentile_high <= 100.0, "percentile should be between 0 and 100"
        assert clip_percentile_low < clip_percentile_high
```
add some reasonable assertions